syntax = "proto3";

message ParentMap {
        map<string, string> my_child = 1;
}

message KeyValue {
        string key = 1;
	string value = 2;
}

message ParentMapFootgun {
	repeated KeyValue my_child = 1;
}
